tutorials/audio/audio_buses.rst

```diff
@@ -162,19 +162,23 @@ Distortion effects make the sound "dirty". Godot offers several types of
 distortion: *overdrive*, *tan* and *bit crushing*. Distortion can be used
 to simulate sound coming through a low-quality speaker or device.
 
-EQ, EQ6, EQ10, EQ21
-~~~~~~~~~~~~~~~~~~~
+EQ
+~~~
 
-Godot provides four equalizers with different numbers of bands. An equalizer on
+EQ is what all other equalizers inherit from and should not be used directly.
```

Comment:
`AudioStreamEQ` can be used to create your own custom equalizer, so this should be documented.